operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -82,8 +82,6 @@ message CNIConfig {
 
   Resources resources = 17;
 
-  google.protobuf.BoolValue privileged = 18;
```

Comment:
Shouldn't we keep this field as reserved so that no one will re-use it in the future with different type?

Reply:
I don't think so. Given how we actually use protos to "wrap" helm charts, just re-using wouldn't work.

But happy to be corrected.

Reply:
I think if we just drop it we will give an error to many users on upgrade.. I wonder if we should just keep it and ignore (we can add a warning it does nothing) rather than fail?

Reply:
just marked is as deprecated in the proto, that should be good enough - helm itself doesn't care, our proto machinery is the only thing that would.